status-client-spec.md

```diff
@@ -78,20 +78,20 @@ We use [`protobuf`](https://developers.google.com/protocol-buffers/) in differen
 
 ### P2P Overlay
 
-Status clients run on the public Ethereum network, as specified by the devP2P
+Status clients run on a private peer-to-peer network, as specified by the devP2P
```

Comment:
Is this accurate? It is permissionless, nothing is stopping anyone from running a node. @corpetty

Reply:
good point, maybe `separate, permissionless` instead of `private`?

Reply:
or just `permissionless`

Reply:
I'd say public and permissionless